x-pack/plugins/security_solution/scripts/endpoint/common/response_actions.ts

```diff
@@ -103,7 +105,8 @@ export const sendEndpointActionResponse = async (
       endpointResponse.EndpointActions.data.output
     ) {
       (
-        endpointResponse.EndpointActions.data.output.content as ResponseActionGetFileOutputContent
+        endpointResponse.EndpointActions.data.output
+          .content as unknown as ResponseActionGetFileOutputContent
```

Comment:
Q: Was casting to `unknown` necessary in these changes to the file? 🤔

Reply:
yes, only because we are not using a Type guard here and because I have hardened the types around the `ActionDetails` structure. Maybe I'll change it to use the type guard in the future.